projects/ngrx.io/content/guide/component/index.md

```diff
@@ -10,29 +12,15 @@ This package is still experimental and may change during development.
 
 ## Key Concepts
 
-Rendering happens in the template only:
-  - The [ngrxPush pipe](guide/component/push) provides a drop-in replacement for the `async` pipe.
-  - The [ngrxLet directive](guide/component/let) provides a structural directive with better support for handling observables. 
+- Rendering observable events in a performant way.
+- Displaying different content based on the current state of an observable.
+- Building fully reactive Angular applications regardless of whether `zone.js` is present or not.
 
 ## Installation
 
 Detailed installation instructions can be found on the [Installation](guide/component/install) page.
 
-## Usage
+## Available Features
 
-Both the `*ngrxLet` directive, and `ngrxPush` pipe are provided through the `ReactiveComponentModule`. To use them, add the `ReactiveComponentModule` to the `imports` of your NgModule.
-
-```typescript
-import { NgModule } from '@angular/core';
-import { ReactiveComponentModule } from '@ngrx/component';
-
-@NgModule({
-  imports: [
-    // other imports
-    ReactiveComponentModule
-  ]
-})
-export class MyFeatureModule {}
-```
-
-Read more about the [ngrxLet directive](guide/component/let), and [ngrxPush pipe](guide/component/push).
+Learn more about features provided by the `@ngrx/component` package through the [ngrxLet directive](guide/component/let)
+and [ngrxPush pipe](guide/component/push) docs.
```

Comment:
For consistency, or don't we want to use code blocks here to make the link more visible?
```suggestion
Learn more about features provided by the `@ngrx/component` package through the [`ngrxLet` directive](guide/component/let)
and [`ngrxPush` pipe](guide/component/push) docs.
```